src/components/Spoiler/Spoiler.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+import { AnimatePresence, motion } from "framer-motion";
+
+import styles from "./Spoiler.module.scss";
+import arrow from "../../assets/svg/arrow.svg";
+
+const Spoiler = ({ children, title }) => {
+  const [isVisible, setVisible] = useState(false);
+
+  return (
+    <div className={styles.spoiler}>
+      <h3 className={styles.spoilerTitle} onClick={() => setVisible(!isVisible)}>
```

Comment:
better to change state based on previous state as a argument in setState function () => setVisible(prev => !prev)